stable/chaoskube/templates/deployment.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if semverCompare "^1.9-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Is it better to use capabilities in https://github.com/helm/helm/blob/master/docs/chart_template_guide/builtin_objects.md: "Capabilities.APIVersions.Has $version indicates whether a version (batch/v1) is enabled on the cluster."